collectors/python.d.plugin/am2320/am320.chart.py

```diff
@@ -0,0 +1,60 @@
+try:
+    import board
+    import busio
+    import adafruit_am2320
+    HAS_STUFF = True
+except ImportError:
+    HAS_STUFF = False
+
+
+#from bases.FrameworkServices.SimpleService import SimpleService
+from base import SimpleService
+
+
+ORDER = [
+    'temperature',
+    'humidity',
+]
+
+CHARTS = {
+    'temperature': {
+        'options': [None, 'Temperature', 'celsius', 'temperature', 'am2320.temperature', 'line'],
+        'lines': [
+            ['temperature']
+        ]
+    },
+    'humidity': {
+        'options': [None, 'Relative Humidity', 'percentage', 'humidity', 'am2320.humidity', 'line'],
+        'lines': [
+            ['humidity']
+        ]
+    }
+}
+
+
+class Service(SimpleService):
+    def __init__(self, configuration=None, name=None):
+        SimpleService.__init__(self, configuration=configuration, name=name)
+        self.order = ORDER
+        self.definitions = CHARTS
+        self.am = None
+
+    def check(self):
+        if not HAS_STUFF:
+            self.error("following libraries are needed to get module working: board, busio, adafruit_am2320")
```

Comment:
This should ideally reference the package that the modules are part of instead of the specific modules, especially since `board` and `busio` are really generic and don't turn up a specific package when searching [PyPI](https://pypi.org/).

Reply:
yes, according docs all user to do is

> sudo pip3 install adafruit-circuitpython-am2320

so call it `AM2320 library`

Reply:
I'd go a bit further and refer to the full package name.  It's not likely that there will be any confusion over the naming, but it's best to just be specific when writing error messages.

Reply:
The full PyPI package name is adafruit-circuitpython-am2320, so the error message should read something like either:

`Could not find the AM2320 library.`

or:

`Could not find the adafruit-circuitpython-am2320 package.`

Reply:
HAS_STUFF -> HAS_AM2320

following libraries are needed -> adafruit-circuitpython-am2320 library is needed